README.md

```diff
@@ -112,7 +112,7 @@ dependencies {
 #### Async mode
 
 * Kotlin
-``` Kotlin
```

Comment:
[Q] Does it make any difference?

Reply:
Yep. Not on GitHub, but it does on compiling jdoc/kdoc, and various plugins in Intelij/Android Studio.